### ISSUE TITLE
In-Game Leaderboard not working in main client

### ISSUE THREAD

**Issue by WindStrike72** [Karawasa/Element-TD#470]
In the main client of the game, the in-game leaderboard apparently doesn't work. Oddly, it works for me, buuuut not everyone, sooooo I got no idea.

**Comment by biax** [comment]
[http://prntscr.com/d0z8qb](url)

**Comment by WindStrike72** [comment]
OOOOOOOH THAT LEADERBOARD, I FORGOT IT EXISTS!

Yeah, it doesn't work period, lol.

**Comment by biax** [comment]
A permanent fix is necessary. I hope @MNoya can share some pointers to avoid this from happening again.

**Comment by MNoya** [comment]
Problem is I never managed to reproduce this problem on my local version. Is this happening on anyone's local version, or only after it's uploaded ingame on the dedicated server?

**Comment by biax** [comment]
Local builds work fine here. Martin for now, could you rebuild/reupload 
the latest master? The last time you did it Leaderboard started working.

**Comment by MNoya** [comment]
Is this still a problem?

**Comment by ynohtna92** [comment]
Yeah it is. I've seen it work occasionally on the workshop but then other times it will not load. I think there is an issue with the http get/post from the client.
